assets/airports/kpvd.json

```diff
@@ -0,0 +1,6509 @@
+        "WEGOT":        ["N41d32m25.3800",  "W072d35m32.1900"],
+        "WERID":        ["N41d13m06.5900",  "W070d46m38.6400"],
+        "WHISL":        ["N41d50m10.6000",  "W072d47m18.5500"],
+        "WIKIT":        ["N41d38m53.4200",  "W071d29m23.9400"],
+        "WILIM":        ["N41d36m59.1000",  "W072d47m22.7800"],
+        "WIPOR":        ["N41d39m23.0700",  "W072d10m37.6700"],
+        "WIVPA":        ["N41d45m10.9900",  "W071d24m07.2500"],
+        "WOMUL":        ["N41d33m17.2400",  "W072d55m50.0100"],
+        "WOONS":        ["N41d57m02.0700",  "W071d30m25.3400"],
+        "XOBIJ":        ["N41d54m08.2200",  "W072d48m48.7500"],
+        "YBERA":        ["N41d38m20.6100",  "W071d31m29.9800"],
+        "YELOW":        ["N41d55m49.0700",  "W072d51m17.7000"],
+        "YODEL":        ["N41d23m25.3700",  "W072d39m30.5200"],
+        "YODER":        ["N41d17m22.5100",  "W072d20m32.7900"],
+        "ZADOR":        ["N41d39m56.9700",  "W072d48m00.3200"],
+        "ZAGAR":        ["N42d00m21.2300",  "W072d03m24.0800"],
+        "ZARAS":        ["N41d43m56.6700",  "W072d14m16.3100"],
+        "ZEPVI":        ["N41d34m33.7300",  "W072d52m43.9100"],
+        "ZOVOX":        ["N41d42m13.0600",  "W072d36m52.9500"],
+        "_MINNK015100": ["N43d01m71.0000", "W071d23m79.000"]
+    },
+    "runways":[
+        {
+            "name": ["05", "23"],
+            "end": [
+                ["N41d42m49.7723",  "W071d26m05.8486", "50.8ft"],
+                ["N41d43m49.6961",  "W071d25m15.5354", "44.3ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["16", "34"],
+            "end":  [
+                ["N41d43m53.9542",  "W071d25m55.8304", "53.7ft"],
+                ["N41d43m06.8846",  "W071d25m05.9925", "32.8ft"]
+            ],
+            "ils": [false, true]
+        }
+    ],
+    "stars": {
+        "JORDN2": {
+            "icao": "JORDN2",
+            "name": "Jordan Two",
+            "entryPoints": {
+                "HTO": ["HTO", "JORDN"]
+            },
+            "body": ["MINNK", ["_MINNK015100"]],
+            "draw": [["HTO*", "JORDN", "MINNK"]]
+        },
+        "WIPOR3": {
+            "icao": "WIPOR3",
+            "name": "Wipor Three",
+            "entryPoints": {
+                "ALB": ["ALB"],
+                "HNK": ["HNK", ["HINGZ", "A290"]],
+                "HINGZ": [["HINGZ", "A290"]]
+            },
+            "body": ["PONEE", ["NELIE", "A210-"], ["WIPOR", "A110"], "ORW", "SAUGY"],
+            "rwy": {
+                "05": ["LAFAY"],
+                "34": ["LAFAY"],
+                "16": ["WAWIK", "SEABE", "NOSUH"],
+                "23": ["WAWIK", "SEABE", "NOSUH"]
+            },
+            "draw": [
+                ["ALB*", "SHIGY", "PONEE"],
+                ["HNK*", "HINGZ*", "PONEE"],
+                ["PONEE", "NELIE", "WIPOR", "ORW", "SAUGY"]
+            ]
+        }
+    },
+    "spawnPatterns": [
```

Comment:
From [the airport master record](http://www.gcr1.com/5010ReportRouter/PVD.pdf), the 2015 stats indicate 66,537 aircraft movements. IAW the [traffic level specs](https://github.com/openscope/openscope/blob/develop/documentation/airport-file-standards.md), this should fall between 7.5acph and 22.8acph. Currently, there's a total of 6acph.

Please increase the traffic level.